src/main.tsx

```diff
@@ -18,12 +19,14 @@ const queryClient = new QueryClient({
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
     <ToggleColorMode>
-      <QueryClientProvider client={queryClient}>
-        <ReactQueryDevtools initialIsOpen={false} />
-        <BrowserRouter>
-          <App />
-        </BrowserRouter>
-      </QueryClientProvider>
+      <NewDeliveryProvider>
```

Comment:
dont put this here. put it where you have the new delivery form. we dont want the whole app to know about this, only the new delivery.

Reply:
I think that I should put newDeliveryProvider in deliveries component but when i put them it is not work. Can you help me?
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/123353314/591f94e8-5aa0-4438-9259-fcaf35702a59)

Reply:
maybe try inside the router or send what error you get?

Reply:
Fixed